tools/integration_tests/rename_dir_limit/move_dir_test.go

```diff
@@ -194,21 +193,22 @@ func TestMoveDirectoryInEmptyDirectory(t *testing.T) {
 }
 
 func createDestNonEmptyDirectoryForMoveTest(t *testing.T) {
```

Comment:
Nit: Is this the only place where we are using testDir? If. yes, we can remove the global variable and pass destDir as parameter to the function. This will also make it consistent with `createSrcDirectoryWithObjectsForMoveDirTest` method.